tests/end2end-tests/graphstorm-er/test.sh

```diff
@@ -7,7 +7,8 @@ GS_HOME=$(pwd)
 NUM_TRAINERS=1
 export PYTHONPATH=$GS_HOME/python/
 cd $GS_HOME/training_scripts/gsgnn_ep
-
+echo "127.0.0.1" > ip_list.txt
+cd $GS_HOME/inference_scripts/ep_infer
```

Comment:
Why change the workspace here?

Reply:
Other end2end tests use two workspaces. One for training and one for inference. And `ip_list.txt` file is created in both location.

But before this PR, there is only one workshop for ER tests. No previous case checks inference. As I need to do inference here, create a new `ip_list.txt` in the inference workspace.